ytfzf

```diff
@@ -468,6 +477,8 @@ start_ueberzug () {
 stop_ueberzug () {
     exec 3>&-
     rm "$FIFO" > /dev/null 2>&1
+    # make sure ueberzug is really dead
+    ps aux | grep 'ueberzug' | awk '{print $2}' | sed '1q' | xargs kill
```

Comment:
the user might have other programs using ueberzug

Reply:
true, not sure how to fix that.

Reply:
I think I should go back to using $!.
@Euro20179 seems to think it does not work in dash, but it works with my dash, so it must be a problem on your end.

Reply:
oh, yes it was a problem with my end.